Polyhedron/demo/Polyhedron/Use_ssh.cpp

```diff
@@ -182,7 +182,7 @@ bool push_file(ssh_session &session,
     return false;
   }
   //push a file to /tmp
-  res = ssh_scp_push_directory(scp, ".", 0707);
+  res = ssh_scp_push_directory(scp, ".", 0777);
```

Comment:
Do you have any idea what 0777 mean?!

It means that everybody on the system can read, write, access to it with `cd`, like in `/tmp`. The problem read is the "write" access: that is a serious security issue!

A safe mode for a directory is 0755 (aka `rwxr-xr-x`): everybody can read and enter the directory,  but not write to it. I will push a commit to `master` directly to fix that issue.